frontend/packages/dev-console/integration-tests/features/web-terminal/web-Terminal-adminuser.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Web Terminal for Developer user
+    As a user with admin rights, I should be able to use web terminal
+
+
+Background:
+    Given user has logged in as admin user
+    And user is at developer perspecitve
+    And user has installed Web Terminal operator
+    And user has created Web Terminal instance
+    And user has created namespace "aut-terminal-admin"
```

Comment:
I think we should remove this line, As In first scenario we are using project creation

Reply:
There are two different scenarios, one is selecting already created namespace and the other one is creating namespace in the terminal window itself. That's why I added that line.

Reply:
Looks to me like the scenario could simply use _any_ namespace to begin with. As the starting namespace is irrelevant if we are creating a new namespace.

Reply:
Okay. I will remove this scenario.